apps/dc_tools/odc/apps/dc_tools/utils.py

```diff
@@ -340,7 +336,7 @@ def item_to_meta_uri(
             "Couldn't find matching product for product name: %s",
             product_name_sanitised,
         )
-        raise SkippedException(
+        raise DatsetExists(
```

Comment:
This is really a mis-use of the documented behavior, but there is a warning emitted in the previous line.